hardware/action_examples/hls_search/hls_search/.project

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do we need this? Did you do a git commit -a without reviewing the files you were adding. Please remove if not needed.